Build/Projects/FrameworkReferences.definition

```diff
@@ -25,6 +29,7 @@
         Name="OpenTK"
         Path="ThirdParty/Dependencies/OpenTK.dll" />
     </Service>
+    <Reference Include="MonoGame.Framework.Core.Include" />
```

Comment:
I meant end users of MonoGame, i.e. The developers using it to build games.

Installing the PCL reference assemblies has come up twice in the monodevelop Gitter channel this week, and this was from people using Debian and Gentoo. Any Linux distribution that mandates the source code must be available for anything they package can't distribute the reference assemblies because they're not open source. That in turn means that on no Linux distros do you get them by installing mono-complete, you have to seperately install the reference-assemblies package. It's a lot of pain for very little benefit, especially since Protobuild makes it pretty easy to just include the files on the Linux platform and avoid any of the support issues that would arise otherwise.